README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
```

Comment:
I have added a PR about enums, but I don't specify which file they should go in, just general usage. I'm wondering if this applies to enums or not.